### ISSUE TITLE
Compiler fails to compile alias qualified name in string interpolation

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#71203]
**Version Used**: SharpLab's main (e8790cc)

**Steps to Reproduce**:

```csharp
```

A minimal repro, with source-code provided, is ideal.  Using [sharplab](https://sharplab.io/) is preferred for compiler/language issues whenever possible.

**Diagnostic Id**:

If this is a report about a bug in an analyzer, please include the diagnostic if possible (e.g. `"IDE0030"`).

**Expected Behavior**:

**Actual Behavior**:

**Comment by 333fred** [comment]
I don't believe this is a bug. If you need to use an expression with a : in it in an interpolation, you need to wrap it in parentheses. @CyrusNajmabadi, does that match your understanding as well?

**Comment by Youssef1313** [comment]
@333fred Wrapping in parentheses work. If that's the expected behavior, I think the error message could be improved though.
